osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SliderBall.cs

```diff
@@ -16,16 +16,30 @@
 using osuTK.Graphics;
 using osu.Game.Skinning;
 using osuTK;
+using osu.Game.Graphics;
 
 namespace osu.Game.Rulesets.Osu.Objects.Drawables.Pieces
 {
-    public class SliderBall : CircularContainer, ISliderProgress, IRequireHighFrequencyMousePosition
+    public class SliderBall : CircularContainer, ISliderProgress, IRequireHighFrequencyMousePosition, IHasAccentColour
     {
         public Func<OsuAction?> GetInitialHitAction;
 
+        private Color4 accentColour;
+
+        public Color4 AccentColour
+        {
+            get => accentColour;
+            set
+            {
+                accentColour = value;
+                ball.Colour = value;
+            }
+        }
```

Comment:
Why is this written in such a convoluted way, instead of:
```suggestion
        public Color4 AccentColour
        {
            get => ball.Colour;
            set => ball.Colour = value;
        }
```
That way (a) reads easier (it's obvious at a glance that it's a delegating property), (b) is less error-prone in case somebody ever exposes `ball` for some reason, (c) gets rid of the field.